main/src/cgeo/geocaching/settings/Settings.java

```diff
@@ -129,14 +129,22 @@ protected Settings() {
         throw new InstantiationError();
     }
 
+    public static int getActualVersion() {
+        return getInt(R.string.pref_settingsversion, 0);
+    }
+
+    public static int getExpectedVersion() {
+        return 4;
```

Comment:
Shouldn't it be rather "5"? That's the current version...

Reply:
actually it is not. if you see the existing migration steps you will see that prop version is never set to 5, but only to 4

Reply:
For me, it actually _is_ the case. Was there some kind of migration routine in the nightly which was removed afterwards? In any case, we should not go back with the version number...

![Screenshot_20210214_120606_copy_536x430](https://user-images.githubusercontent.com/64581222/107874934-59d5f300-6ebd-11eb-9e06-7520630d55a7.png)

Reply:
Ah now that you mention it: yes there was a version 5 spilling into the nightlies for a few days. Should never affect the real users. But as a service to the nightly users I will introduce a dummy version 5 with no migration work. I will push this any minute.

Reply:
Thank you! That's probably the best, since we have a few dozen nightly users...